demo/config/packages/ai.yaml

```diff
@@ -4,34 +4,27 @@ ai:
             api_key: '%env(OPENAI_API_KEY)%'
     agent:
         blog:
-            model:
-                class: 'Symfony\AI\Platform\Bridge\OpenAi\Gpt'
-                name: !php/const Symfony\AI\Platform\Bridge\OpenAi\Gpt::GPT_4O_MINI
+            model: 'gpt-4o-mini'
             tools:
                 - 'Symfony\AI\Agent\Toolbox\Tool\SimilaritySearch'
                 - service: 'clock'
                   name: 'clock'
                   description: 'Provides the current date and time.'
                   method: 'now'
         stream:
-            model:
-                class: 'Symfony\AI\Platform\Bridge\OpenAi\Gpt'
-                name: !php/const Symfony\AI\Platform\Bridge\OpenAi\Gpt::GPT_4O_MINI
+            model: 'gpt-4o-mini'
             prompt: |
                 You are an example chat application where messages from the LLM are streamed to the user using
                 Server-Sent Events via `symfony/ux-turbo` / Turbo Streams. This example does not use any custom
                 javascript and solely relies on the built-in `live` & `turbo_stream` Stimulus controllers.
                 Whatever the user asks, tell them about the application & used technologies.
             tools: false
         youtube:
-            model:
-                class: 'Symfony\AI\Platform\Bridge\OpenAi\Gpt'
-                name: !php/const Symfony\AI\Platform\Bridge\OpenAi\Gpt::GPT_4O_MINI
+            model: 'gpt-4o-mini'
             tools: false
         wikipedia:
             model:
-                class: 'Symfony\AI\Platform\Bridge\OpenAi\Gpt'
-                name: !php/const Symfony\AI\Platform\Bridge\OpenAi\Gpt::GPT_4O_MINI
+                name: 'gpt-4o-mini'
                 options:
```

Comment:
make sure this is still working

Reply:
yea i guess that's where the options are used on top of the model - and in general it's a bit more handy if one uses one model instance with multiple platform calls